chromium_src/storage/browser/blob/blob_url_store_impl.cc

```diff
@@ -54,4 +48,12 @@ void BlobURLStoreImpl::ResolveForNavigation(
                                                       std::move(callback));
 }
 
+bool BlobURLStoreImpl::IsBlobResolvable(const GURL& url) const {
+  // Check if the URL is mapped to a BlobURLStore with the current
+  // `storage_key_` or if it's an extension-generated blob.
+  return (registry_ && registry_->IsUrlMapped(url, storage_key_)) ||
+         (url.SchemeIsBlob() &&
+          url::Origin::Create(url).scheme() == "chrome-extension");
```

Comment:
Does it match Chromium where this URL is available only in the ISOLATED or USER_SCRIPT world of the extension's content script and not in the MAIN world of the page or in another extension's isolated world?

Reply:
> Does it match Chromium where this URL is available only in the ISOLATED or USER_SCRIPT world of the extension's content script and not in the MAIN world of the page or in another extension's isolated world?

This restriction I'm modifying here has nothing to do with other Chromium checks. They all are in place and should work. Here I'm only disabling *storage partitioning* check for `chrome-extension` blobs.